README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# SUBSCRIPTIONS CUSTOMER PORTAL
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Local Set-Up
 
-## Available Scripts
+### Create a GitHub account if you do not have one already
 
-In the project directory, you can run:
+- https://github.com/
```

Comment:
I think it can be assumed that someone has already done this and/or knows how. Let's delete this step.